test/button/button-subtle-perceptual.js

```diff
@@ -0,0 +1,67 @@
+const puppeteer = require('puppeteer');
+const visualDiff = require('visual-diff');
+
+visualDiff.run((ctx) => {
+
+	describe('d2l-button-subtle', function() {
+
+		this.timeout(10000);
```

Comment:
What's this timeout for?

Reply:
Seems like the default timeout (2s) is not enough - without it, the tests sometimes timeout... not always though.

Reply:
Oh I see, it's not actually waiting 10 seconds, it's just giving the pages at least 10 seconds to do their thing.

Reply:
Yep... it's the mocha timeout.  There might be a way to specify it on the CLI instead, which may be preferred as to avoid setting it in each test file.